numpy/core/include/numpy/npy_common.h

```diff
@@ -262,11 +262,11 @@ typedef Py_uintptr_t npy_uintp;
 #define constchar char
 
 /* NPY_INTP_FMT Note:
- *      Unlike the other NPY_*_FMT macros which are used with
- *      PyOS_snprintf, NPY_INTP_FMT is used with PyErr_Format and
- *      PyString_Format. These functions use different formatting
- *      codes which are portably specified according to the Python
- *      documentation. See ticket #1795.
+ *      Unlike the other NPY_*_FMT macros which are used with PyOS_snprintf,
+ *      NPY_INTP_FMT is used with PyErr_Format, PyUnicode_Format, and
```

Comment:
`PyUnicode_Format` is a red herring here, that uses python format-codes not C ones.

Reply:
Fixed. Now that `npy_intp` is defined as `Py_ssize_t` we could probably clean up some of the uses.